.github/workflows/release.yaml

```diff
@@ -57,13 +60,19 @@ jobs:
 
     - name: Draft release
       id: draft_release
+      # TODO: This action is no longer mainained. We should use a different action
+      # or the gh command line directly.
```

Comment:
Can you open an issue instead or in addition?

Reply:
Done: https://github.com/shipwright-io/build/issues/1567